### PR TITLE
HW#1_Joanna_Zhang 

### PR DESCRIPTION
**Pull request by joannazhang88**
I like the way that this assignment created a chance for us to know each other by reading other students' hw. I first had some trouble with the fork-push-pull process, but our Mint and Xiaogao helped out. And I think going over the 'using Git in RStudio' section on our course website would be very helpful before starting this hw, because sometimes RStudio just somehow couldn't find Git.

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* You completed all the requirements for the HW. Great job :)
* And the picture work 💯

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* You completed all the requirements for the HW. Great job :)
* And the picture work 💯

**Review by atani21 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Good work! I like the way the different sections are separated. It makes the whole page easier to read.

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Hello！
+
+## Brief Intro
+My name is **Joanna Zhang**. I'm a MAPSS student with an economics concentration. I spent my undergrad years at Oberlin College. This is my first year at UChicago so I'm very excited about meeting new people! You can find my LinkedIn page [here](https://www.linkedin.com/in/jiahang-zhang-aa858113b/).
```
Comment (wanitchayap) [inline README.md:6]: Great code on embedding the link!

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+## Research Interests
+I am interested in the global financial markets. Specifically....
+
+* Asset Pricing 
+* Emerging Markets
+* Blockchain & Cryptocurrency
+We can talk if you're interested in one of those!
+
+And finally a picture of me. ![my picture](dc.jpeg)
```
Comment (wanitchayap) [inline README.md:16]: Might be better to put the picture code chuck into a new line.

**Pull request closed without merging**